frontend/docsettings.lua

```diff
@@ -10,12 +10,14 @@ local dump = require("dump")
 local ffiutil = require("ffi/util")
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
+local document = require("document.document")
```

Comment:
We never really use the automagic path separator syntax, you can just enforce a / here.

Reply:
yeah not after move to util. Done